t/t205-event-locking.t

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+#
+# Copyright (C) 2018-2019 GitHub, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses/ .
+
+test_description='projfs file operation locking tests
+
+Check that projfs file operation notification events are issued serially for a
+given path.
+'
+
+. ./test-lib.sh
+. "$TEST_DIRECTORY"/test-lib-event.sh
+
+rm lock >/dev/null 2>/dev/null
```

Comment:
And I also think you can probably skip the `rm` here because the tests always run in a fresh subdir under `./test-mounts` and so there shouldn't be any unexpected files lurking around, unless I'm mistaken (and if I am, and you need the `rm`, would `rm -f` suffice?)

Reply:
You are most certainly not. 👍